.github/workflows/rust.yml

```
@@ -99,6 +99,11 @@ jobs:
           rust-version: stable
       - name: Run tests (excluding doctests)
         run: cargo test --lib --tests --bins --features avro,json,backtrace
+        env:
+          # do not produce debug symbols to keep memory usage down
+          RUSTFLAGS: "-C debuginfo=0"
+          RUST_BACKTRACE: "1"
+          RUST_MIN_STACK: "2384500"
```

I think it would be good to add context to why the `RUST_MIN_STACK` is set to help us in the future remember 

```suggestion
          # avoid  thread 'tpcds_physical_q54' has overflowed its stack
          RUST_MIN_STACK: "2384500"
```

(and the same comment applies to the other places in this file)